axes/apps.py

```diff
@@ -1,16 +1,28 @@
+from __future__ import absolute_import
+
+import logging
+
 from django import apps
 
+from .conf import settings
+
+
+log = logging.getLogger(settings.AXES_LOGGER)
+
 
 class AppConfig(apps.AppConfig):
     name = 'axes'
 
     def ready(self):
-        from django.contrib.auth.views import LoginView
-        from django.utils.decorators import method_decorator
+        from . import signals    # NOQA: Load the signals module to bind the receivers
+
+        if settings.USE_LEGACY_LOGIN_VIEW_PATCH:
+            log.warning('Automatic patching of LoginView using AXES_USE_LEGACY_LOGIN_VIEW_PATCH is deprecated. '
```

Comment:
You should use DeprecationWarning for this.

https://docs.python.org/3.6/library/exceptions.html#DeprecationWarning